blivet/formats/fs.py

```diff
@@ -765,6 +765,18 @@ def unmount(self, **kwargs):
         """
         return self.teardown(**kwargs)
 
+    def umount(self, **kwargs):
+        """ Unmount this filesystem.
+
+            :param str mountpoint: Optional mountpoint to be unmounted.
```

Comment:
While doing this, could you please replace ``param`` with ``keyword`` here as well as in the ``unmount()`` method? That's the proper field name, I think.